### ISSUE TITLE
@JSON cant have negative number

### ISSUE THREAD

**Issue by mathiasrw** [agershun/alasql#475]
When having minus in front of a (number) value the parser cannot handle the JSON

```
> alasql "SELECT * FROM @[{delay:5},{delay:-7}]"
[Error: Parse error on line 1:
...M @[{delay:5},{delay:-7}]
-----------------------^
Expecting 'COLON', got 'COLONDASH']
```

Workaound in the moment: put `'` around the negative numbers (`'-7'`)

**Comment by agershun** [comment]
@mathiasrw Yes,now you need to put ```()``` around any expression inside JSON object. This is a  parser problem, because it does not know how to handle this expression:
```
    @{a:@{b}}
```
is this the expression inside expression or this is a sub-json object. Probably, we have to leave it as it is.

**Comment by mathiasrw** [comment]
Good to see you again. 

Lets leave it as it is for the moment. 

I will update all examples to put  `'` around so we dont get an issue.

**Comment by mathiasrw** [comment]
Fix released as part of v4.12.0
